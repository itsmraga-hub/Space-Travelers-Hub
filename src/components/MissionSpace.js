import style from './css/MissionSpace.module.css';

const MissionSpace = (props) => {
  const mission = props;
  const { name, id } = mission;
  return (
    <li id={id} className={style.Mission}>{name}</li>
  );
};

export default MissionSpace;
