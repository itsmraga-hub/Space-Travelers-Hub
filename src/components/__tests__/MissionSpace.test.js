import renderer from 'react-test-renderer';

import MissionSpace from '../MissionSpace';

const mission = {
  name: 'SpaceX',
  id: '87GHTR6',
};

it('<Mission /> renders properly as expected', () => {
  const {
    name, id,
  } = mission;
  const tree = renderer.create(<MissionSpace
    id={id}
    name={name}
  />).toJSON();
  expect(tree).toMatchSnapshot();
});
