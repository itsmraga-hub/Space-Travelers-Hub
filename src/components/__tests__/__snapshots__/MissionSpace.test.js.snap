// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Mission /> renders properly as expected 1`] = `
<li
  className="Mission"
  id="87GHTR6"
>
  SpaceX
</li>
`;
